katello/katello-backup

```diff
@@ -1,94 +1,113 @@
 #!/bin/bash
 
 usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
+  echo "Usage: katello-backup --directory <backup directory>"
+  echo " eg: $ katello-backup --directory /tmp/katello-backup/"
```

Comment:
The point that I am attempting (poorly) to get across is that arguments are for required information that a CLI tool needs, and options are, well, optional.

I realize that Hammer uses options for both required and non-required information completely throwing out the entire notion of positional arguments which drives me bonkers, but that ship has already sailed and I doubt anyone would hear me out to change that now.

If you're wanting to use `--directory DIR` instead of requiring a positional `/path/to/dir` argument for the sake of using the same style as Hammer, then I can live with that.